backend/migrations/000014_add_one_address_strategy.down.sql

```diff
@@ -0,0 +1 @@
+DELETE FROM voting_strategies WHERE key='one-address-one-vote' 
```

Comment:
FYI you wont need to worry about remove enum here because removing single type from enum in postgres is not supported (just keep this file as is!)